src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -6431,7 +6431,7 @@ def test_aks_create_with_azuremonitormetrics(self, resource_group, resource_grou
         })
 
         create_cmd = 'aks create --resource-group={resource_group} --name={name} --location={location} --ssh-key-value={ssh_key_value} --node-vm-size={node_vm_size} ' \
-                     '--enable-managed-identity --enable-azuremonitormetrics --enable-windows-recording-rules --output=json'
+                     '--enable-managed-identity --enable-azure-monitor-metrics --enable-windows-recording-rules --output=json'
```

Comment:
Queued [live test pipeline](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=74513945&view=results) to validate the touched test cases.